src/common/components/cards/rich-resolution-content.tsx

```diff
@@ -89,6 +89,174 @@ export const RichResolutionContent = NamedFC<RichResolutionContentProps>(
                     </span>
                 );
             }
+            case 'android/atfa/ClassNameCheck': {
+                return (
+                    <span>
+                        Provide a <Markup.Code>className</Markup.Code> value within the element's{' '}
+                        <LinkComponent href="https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html">
+                            AccessibilityNodeInfo
+                        </LinkComponent>{' '}
+                        that:
+                        <ol>
+                            <li>Closely describes the element's function,</li>
+                            <li>Matches a class that extends android.view.View,</li>
+                            <li>Is provided with the Android SDK or support libraries, and</li>
+                            <li>Is as far down Android's class hierarchy as possible.</li>
+                        </ol>
+                        Note: Do not attempt to fix this issue by adding the class name to the
+                        object's ContentDescription, as that approach can cause some assistive
```

Comment:
Referencing the same doc
```suggestion
                        object's <Markup.Code>ContentDescription</Markup.Code>, as that approach can cause some assistive
```

Reply:
Nice catch!